src/_help/bump-cli.md

```diff
@@ -28,164 +35,205 @@ Or, with Yarn via
 
-You can also preview a file available from a URL
+ You can also get some help anytime by adding `--help` to any command. Example: `bump deploy --help`.
+
+## Prepare your Bump.sh account
+
+While some commands don't need any API token (`preview` or `diff`) you will need an access key if you want to interact with your Bump.sh documentation.
+
+Head to your Documentation settings in the “CI deployment” section, or your Account or Organization settings in the “API keys” section to fetch a personal token for later usage.
+
+## Commands
+
+* [`bump deploy [FILE]`](#bump-deploy-file)
+* [`bump diff [FILE]`](#bump-diff-file)
+* [`bump preview [FILE]`](#bump-preview-file)
+
+### `bump deploy [FILE]`
+
+When you update your API, you want its documentation to be live for your API users. This is what the deploy command is for.
 
 ```shell
-bump preview https://developers.bump.sh/source.yaml
+bump deploy path/to/api-document.yml --doc my-documentation --token $DOC_TOKEN
 ```
 
-### Live preview
+> You can find your own `my-documentation` slug and `$DOC_TOKEN` api key from your [documentation settings](https://bump.sh/docs).
+{: .info}
 
-By using the `--live` flag you can stay focused on API design (OpenAPI or AsyncAPI file) while seeing a continuously updated preview each time you save your API definition file.
-
-- Launch the live preview command in your terminal
+You can also deploy a given API document to a different branch of your documentation with the `--branch <branch-name>` parameter. Please note the branch will be created if it doesn’t exist. More details about the branching feature is available on [this dedicated help page](https://docs.bump.sh/help/branching). E.g. deploy the API document to the `staging` branch of the documentation:
 
 ```shell
-bump preview --live --open openapi-definition.json
+bump deploy path/to/api-document.yml --doc my-documentation --token $DOC_TOKEN --branch staging
 ```
 
-- Edit your `openapi-definition.json` file in your favorite text editor
-- Watch the live preview being updated each time your save your file!
+#### Deploy a folder all at once
 
-> You can create as many previews as you like without being authenticated. This is a **free and unlimited service**.
+If you already have a hub in your [Bump.sh](https://bump.sh) account, you can automatically create documentation inside it and deploy to it by publishing a whole directory containing multiple API documents in a single command
+
+```shell
+bump deploy dir/path/to/apis/ --auto-create --hub my-hub --token $HUB_TOKEN
+```
+
+> You can find your own `my-hub` slug and `$HUB_TOKEN` api key from your [hub settings](https://bump.sh/hubs).
 {: .info}
 
-## API diff or simple validation before a deployment
+Please note, by default, only files named `{slug}-api.[format]` are published. Where `{slug}` is a name for your API and `[format]` is either `yaml` or `json`. Take into account your own file naming convention by using the `--filename-pattern <pattern>` option.
```

Comment:
```suggestion
Please note that, by default, only files named `{slug}-api.[format]` are published. Where `{slug}` is a name for your API and `[format]` is either `yaml` or `json`. Adjust to your file naming convention using the `--filename-pattern <pattern>` option.
```

Reply:
I'm not a huge fan of using `that` it's usually an optional word and doesn't help that much in this context imo.